### ISSUE TITLE
Help - How to add Netbeans for UserLAnd (Application screen)?

### ISSUE THREAD

**Issue by foodmaher** [CypherpunkArmory/UserLAnd#715]
# Describe the issue you are having.

This should include things like:
- I want to add Netbeans to the application screen so that I can use it quickly and also be available to the user to try it.
- I didn't try anything because I don't know how to add programs to your application.

# Describe how your task behaves in a regular Linux environment
Behave normally.

But I want to know how to add programs to the application interface, what files should be modified and how can I implement a bash script and link it to the application so that it executes when pressed to install the program because, as you know Netbeans needs to install Java and other accessories.

**Note:** Does UserLAnd support GLX 1.3 OR UP OpenGL for (JavaFX)?

        Termux screen

![Screenshot_2019-03-22-13-37-23](https://user-images.githubusercontent.com/27397889/54820604-7bfe7080-4ca8-11e9-9a8e-587ba9fa763f.png)
![Screenshot_2019-03-22-13-38-48](https://user-images.githubusercontent.com/27397889/54820616-828ce800-4ca8-11e9-9461-c70383449b34.png)
![Screenshot_2019-03-22-13-42-06](https://user-images.githubusercontent.com/27397889/54820625-87519c00-4ca8-11e9-9bd7-d8cecedb0312.png)

**Edit:**  I have a problem `Client is not authorized to connect to Server` When I try to run Netbeans on UserLAnd only.

**Comment by Marcus0086** [comment]
@foodmaher is netbeans working properly?

**Comment by foodmaher** [comment]
@Marcus0086 Yes works correctly but not on Arch Linux distribution, The problem is that the Linux distributions available on the application (except Arch Linux) do not support GLX or OpenGL so if you want to use OpenGL libraries, it will not work with you except Arch Linux because it supports OpenGL.

**Note:** You will need a good specification to run such programs as it consumes between 100 to 500 or more of the RAM

**Comment by corbinlc** [comment]
Closing as part of a repository cleanup. If this issue is still relevant, feel free to reopen it or file a fresh one.
